### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by gcapizzi**
## Is there a related GitHub Issue?

No.

## What is this change about?

This improves our docs in various ways, removing redundancy and leveraging our new Helm chart.

## Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -167,17 +128,15 @@ Self-signed TLS certificates are generated automatically by the installation if
 
 If you want to generate certificates yourself, you should not set the `global.generateIngressCertificates` value, and instead provide your certificates to Korifi by creating two TLS secrets:
 
-1. `korifi-api-ingress-cert` in the `korifi-api-system` namespace, and
-2. `korifi-workloads-ingress-cert` in the `korifi-controllers-system` namespace
-
-with the appropriate values.
+1. `korifi-api-ingress-cert`;
```
Comment (georgethebeatle) [inline INSTALL.md:131]: The namespace where the secret shoud be created must be pointed out.

File: INSTALL.md [hunk 2]
```diff
@@ -167,17 +128,15 @@ Self-signed TLS certificates are generated automatically by the installation if
...
+1. `korifi-workloads-ingress-cert`.
```
Comment (georgethebeatle) [inline INSTALL.md:132]: Same as above

File: README.helm.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Korifi Helm chart
+
+This documents the [Helm](https://helm.sh/) chart for [Korifi](https://github.com/cloudfoundry/korifi).
+
+The chart is a composition of subcharts, one per component, with each individual component configuration nested under a top-level key named after the component itself.
+Values under the top-level `global` key apply to all components.
+Each component can be excluded from the deployment by the setting its `include` value to `false`.
+See [_Customizing the Chart Before Installing_](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing) for details on how to specify values when installing a Helm chart.
+
+Here are all the values that can be set for the chart:
```
Comment (georgethebeatle) [inline README.helm.md:10]: Other projects seem to be documenting their helm values in tabular form: https://github.com/concourse/concourse-chart. It looks a bit more organized, not sure if we want to do that

**Review by georgethebeatle — APPROVED** [review]

File: README.helm.md (hunk 3 above)
Comment (gcapizzi) [reply README.helm.md:10]: That looks a bit less readable to me as the table doesn't fit in the page. I'd stick with this format for now and see how it goes.

**Pull request merged**